phpstan.neon

```diff
@@ -1,71 +1,6 @@
 parameters:
-    bootstrap: %rootDir%/../../../phpstan-bootstrap.php
-
+    checkGenericClassInNonGenericObjectType: false
+    checkMissingIterableValueType: false
     ignoreErrors:
-    - '#Call to an undefined method .+LocatorLocatorInterface::.+\(\).#'
-
-services:
-    -
-        class: PhpStan\DynamicType\FacadeDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\QueryContainerDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ConsoleDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ControllerDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\PluginDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ServiceDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ConfigDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ClientDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ClientPluginDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\YvesFormTypeDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ZedFormTypeDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\YvesFactoryDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
-
-    -
-        class: PhpStan\DynamicType\ZedFactoryDynamicTypeExtension
-        tags:
-            - phpstan.broker.dynamicMethodReturnTypeExtension
+        - '#.+Generated\\Shared\\Transfer.+.#'
```

Comment:
it is very common pattern we should avoid those. use more specifics patterns here.

Reply:
If there are no transfers generated this is fine for now

But I would advise to generate the transfers as per standalone testing guide
This will be better long term, but can also be a follow up PR.